### PR TITLE
[NG] Add NBS data

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

Adding [Nigerian National Bureau of Statistic Data](https://docs.google.com/spreadsheets/u/2/d/194VHhgfyR6LpJF4DTGVfuLS-B82lQiux75NryD_nUYM/edit#gid=0) to **HURUmap-ng** First Version

- [x] Format all data tables to HURUmap format
- [x] Upload all data tables to HURumap Land
- [x] Add function to profile builder and display to profile detail templates
Fixes # (issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots
Screenshot from 2019-06-24 12-49-28
Screenshot from 2019-06-24 12-48-47

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: hurumap_ng/sql/suspects_prosecuted.sql [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+state	37	2016	2012	23
+state	37	2016	2013	42
+state	37	2016	2014	36
+state	37	2016	2015	41
+state	37	2016	2016	56
+country	NG	2016	2012	2292
+country	NG	2016	2012	1772
+country	NG	2016	2012	1981
+country	NG	2016	2012	1633
+country	NG	2016	2012	2194
```
Comment (kilemensi) [inline hurumap_ng/sql/suspects_prosecuted.sql]: Year `2012` repeated 5 times causing `could not create unique index "suspects_prosecuted_pkey"` error. You probably meant `2012, 2013, 2014, 2015, and 2016`

File: hurumap_ng/sql/food_price.sql [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+ALTER TABLE IF EXISTS ONLY public.faac DROP CONSTRAINT IF EXISTS faac_pkey;
```
Comment (kilemensi) [inline hurumap_ng/sql/food_price.sql]: This file is wrongly exported. It's talking about `faac` but it should be `food_price`.

**Review by kilemensi — CHANGES_REQUESTED** [review]
Looks good @KhadijaMahanga... just a few SQL errors.

**Comment by kilemensi** [comment]
And just one general comment/question @KhadijaMahanga:

Why do we need **two** methods to fetch data for `2016` and `2018`? Aren't these just different release years of the same data?

**Comment by KhadijaMahanga** [comment]
> And just one general comment/question @KhadijaMahanga:
> 
> Why do we need **two** methods to fetch data for `2016` and `2018`? Aren't these just different release years of the same data?

At the moment, there are of different tables for `2016` data and `2018`. Maybe as the data come in this will change. So I hid releases on front end  for now

**Comment by kilemensi** [comment]
> At the moment, there are of different tables for `2016` data and `2018`. Maybe as the data come in this will change. So I hid releases on front end for now

Everybody is just hiding releases! What has releases ever done to anybody?

Cool but since everything comes from NBS, I think releases would have been perfect here.

**Review by kilemensi — APPROVED** [review]
:+1: @KhadijaMahanga

**Pull request merged**